pkg/kubelet/kubelet.go

```diff
@@ -2084,6 +2108,10 @@ func (kl *Kubelet) ResyncInterval() time.Duration {
 
 // ListenAndServe runs the kubelet HTTP server.
 func (kl *Kubelet) ListenAndServe(address net.IP, port uint, tlsOptions *server.TLSOptions, auth server.AuthInterface, enableDebuggingHandlers bool) {
+	if tlsOptions != nil {
+		glog.Infof("Enabling rotating certificates.")
+		tlsOptions.Config.GetCertificate = kl.certificateManager.GetCertificate
```

Comment:
Is this using the same certificate for both the client and server? I think that we want those certs do be different. Maybe use two cert managers or have a cert manager manage multiple certs?